main/pipeline/pipeline.cc

```diff
@@ -640,6 +640,7 @@ ast::ParsedFilesOrCancelled indexSuppliedFiles(core::GlobalState &baseGs, absl::
     }
 
     std::shared_ptr<const core::GlobalState> emptyGs = baseGs.copyForIndex(
+        opts.cacheSensitiveOptions.stripePackages || opts.packageRBIGeneration,
```

Comment:
This ` || opts.packageRBIGeneration` is a bit odd, since we explicity check that `--stripe-packages` is not passed when in package RBI generation, but without the `|| ...`, package RBI generation fails.

My mental model is that we don't want to run the "regular pipeline" version of package, but when running other code, we want to pretend that packager is enabled. Maybe a sign that we need to split the flag somehow, since it's overloaded with 2 meaning right now.

Reply:
This might be another argument for removing rbi generation.